src/ansys/scade/wux/impl/sdyext.py

```diff
@@ -414,8 +414,20 @@ def gen_suite_display_connection(self, f: TextIOBase, ip, output, input):
                         input_subelement,
                     )
             else:
-                field = re.search(r'([a-zA-Z_]+[a-zA-Z_0-9]*) : .*', input.type).group(1)
-                input_c = 'kcg_assign(&{}_L_{}()->{}'.format(input_spec.prefix, input.layer, field)
+                m = re.search(r'([a-zA-Z_]+[a-zA-Z_0-9]*) : .*', input.type)
```

Comment:
I guess this `m` stands for `matches`. Maybe, we could make this name more explicit.